tasks/configure_nginx.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: copy ssl cert
+  copy:
+    content: "{{ DALITE_SSL_CERT }}"
+    dest: "/etc/ssl/private/ssl-cert.pem"
+    group: www-data
+    mode: "740"
+
+- name: copy ssl key
+  copy:
+    content: "{{ DALITE_SSL_KEY }}"
+    dest: "/etc/ssl/private/ssl-cert.key"
+    group: www-data
+    mode: "740"
```

Comment:
@jbzdak No, it doesn't: http://serverfault.com/questions/259302/best-location-for-ssl-certificate-and-private-keys-on-ubuntu

You could have double-checked this before answering?